indico/modules/events/papers/controllers/management.py

```diff
@@ -51,3 +54,32 @@ def _process(self):
             logger.info("Paper managers of %r updated by %r", self.event_new, session.user)
             return jsonify_data(flash=False)
         return jsonify_form(form)
+
+
+class RHSwitchReviewingType(RHManagePapersBase):
+
+    CSRF_ENABLED = True
```

Comment:
This is already inherited from `RHPapersBase` which is a parent of `RHManagePapersBase`.